<?xml version="1.0"?>
<sdf version="1.9">
  <model name="point_lidar">
    <link name="point_lidar">
      <inertial>
        <mass>0.005</mass>
        <inertia>
          <ixx>8.33e-06</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>8.33e-06</iyy>
          <iyz>0</iyz>
          <izz>8.33e-06</izz>
        </inertia>
      </inertial>
      <sensor name="gpu_ray" type="gpu_ray">
        <update_rate>20</update_rate>
        <lidar>
          <scan>
            <horizontal>
              <samples>2</samples>
              <resolution>1</resolution>
              <min_angle>0.0</min_angle>
              <max_angle>0.0</max_angle>
            </horizontal>
          </scan>
          <range>
            <min>0.05</min>
            <max>40</max>
            <resolution>0.01</resolution>
          </range>
          <noise>
            <type>gaussian</type>
            <mean>0</mean>
            <stddev>0.01</stddev>
          </noise>
          <visibility_mask>55</visibility_mask>
        </lidar>
      </sensor>
    </link>
    <frame name="mount_point"/>
  </model>
</sdf>
